cli/src/commands/branch.rs

```diff
@@ -569,6 +569,55 @@ fn cmd_branch_track(
             names.len()
         )?;
     }
+
+    //show conflicted branches if there are some
+    let template = {
+        let language = workspace_command.commit_template_language()?;
+        let text = command
+            .settings()
+            .config()
+            .get::<String>("templates.branch_list")?;
+        workspace_command
+            .parse_template(&language, &text, CommitTemplateLanguage::wrap_ref_name)?
+            .labeled("branch_list")
+    };
```

Comment:
nit: perhaps, this can be moved inside `if let Some(mut formatter) ..` so that template wouldn't have to be parsed on `--quiet`.